index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Shaker Abady Protfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
 </head>
 <body>
     <section class="container">
```

Comment:
- [ ] [optional] Kindy add more size to your logo and hamburger menu, also the hamburger menu color doesn't match the design given.